### PR TITLE
ICU-613 Allow clicking post timestamps on desktop web view

### PR DESCRIPTION
**Pull request by hmhealey**
We were using Utils.isMobile before which only checks screen size instead of UserAgent.isMobile which checks the device

#### Ticket Link
https://mattermost.atlassian.net/browse/ICU-613

#### Checklist
- Has UI changes

### CODE REVIEW

File: components/post_view/post_time.jsx [hunk 1]
```diff
@@ -7,7 +7,8 @@ import {Link} from 'react-router';
 
 import TeamStore from 'stores/team_store.jsx';
 
-import {isMobile, getWindowDimensions} from 'utils/utils.jsx';
+import {isMobile} from 'utils/user_agent.jsx';
+import {getWindowDimensions} from 'utils/utils.jsx';
```
Comment (sudheerDev) [inline components/post_view/post_time.jsx]: getWindowDimensions is used for determining mobile view on resize. If we are going to use user agent i think its safe to remove `getWindowDimensions ` now.

Comment (stephenkiers) [reply components/post_view/post_time.jsx]: Yeah. I am confused as to what getWindowDimensions is even used for in this file? It sets the state, but I don't see the state being used at all...

Comment (stephenkiers) [reply components/post_view/post_time.jsx]: Although that isn't your change, I wonder if you should remove it since you are in the file already?

Comment (sudheerDev) [reply components/post_view/post_time.jsx]: @stephenkiers it is just for [re-rendering](https://github.com/mattermost/mattermost-webapp/pull/435#discussion_r156167921) component on resize

Comment (stephenkiers) [reply components/post_view/post_time.jsx]: So, it is used to trigger a setState, and hence a re-render, when the window size changes? ok...

Comment (sudheerDev) [reply components/post_view/post_time.jsx]: Yes. From ticket looks like this behaviour is not intended now so, was suggesting to remove

**Comment by hmhealey** [comment]
Great catch, @sudheerDev. I've removed that

**Comment by sudheerDev** [comment]
@hmhealey Thanks. BTW not to push you on this but if you look at this [PR](https://github.com/mattermost/mattermost-webapp/pull/629) you can find instances where this is being used. They are not needed anymore if you want to remove them as well. If there is another ticket i can have a look at it as well

**Comment by hmhealey** [comment]
@sudheerDev Cool, I'll remove those as well

**Comment by hmhealey** [comment]
Changes made to have all the post-like components use PostTime as per Sudheer's recommendation. It's a pretty big change, so I flagged this for re-review

**Comment by saturninoabril** [comment]
Just need to fix merge conflicts.

**Pull request merged**